src/bp-templates/bp-nouveau/includes/activity/ajax.php

```diff
@@ -703,6 +703,12 @@ function bp_nouveau_ajax_post_update() {
 
 				$is_private = 'public' !== $status;
 			}
+		} else {
```

Comment:
@bb-yudhisthir I don't think this is correct. what if group component disabled

Reply:
@bb-yudhisthir  
`if ( $item_id && bp_is_active( 'groups' ) ) { ` 
and 
`if ( bp_is_active( 'groups' ) ) {
if ( ! $item_id ){`

both are same condition.